server/apps/content.py

```diff
@@ -62,6 +57,15 @@
         'type': 'string',
         'mapping': not_analyzed
     },
+    'event_id': {
```

Comment:
We place archive specific properties in `archive/common.py#item_schema()` https://github.com/superdesk/superdesk/blob/master/server/apps/archive/common.py#L332